docs/creating_build_vm.txt

```diff
@@ -71,9 +69,16 @@ Create /build
   /kickstarts
   /logs
   /references
-  /scripts
   /storage
 
+Clone the build scripts
+
+  cd /build
+  git clone git@github.com:manageiq/manageiq-appliance-build.git
```

Comment:
oh man, sorry...

we probably want to do this:
```
git clone -o upstream https://www.github.com/ManageIQ/manageiq-appliance-build.git
```

This way we avoid needing ssh keys on imagefactory vm, I think.
And, if you have a fork of the manageiq-appliance-build repo, you can add your origin.